manifests/fragment.pp

```diff
@@ -40,9 +40,9 @@
     default => $serverversion,
   }
   if versioncmp($clientversion, '6.0') >= 0 and versioncmp($_serverversion, '6.0') >= 0 {
-    assert_type(Optional[Variant[String, Deferred]], $content)
+    assert_type(Optional[Variant[Sensitive[String], String, Deferred]], $content)
   } else {
-    assert_type(Optional[String], $content)
+    assert_type(Optional[Sensitive[String], String], $content)
```

Comment:
Did Puppet 5 have a `Sensitive` type? Though I also wonder if by now it couldn't be dropped.

Reply:
Puppet 5 is still a thing? :D But, yes. It seems Sensitive was already support in Puppet 5 -> https://www.puppet.com/docs/puppet/5.5/lang_data_sensitive.html

Reply:
The reason I mention it is that's what the `if` statement above is about.

Reply:
the module claims to not support [puppet < 6](https://github.com/puppetlabs/puppetlabs-concat/blob/main/metadata.json#L105) so i wonder if we just drop that if statement

Reply:
Have nice one! :) I love to remove code I've added ages ago :) brb